setup.py

```diff
@@ -78,6 +78,13 @@
     packages=['markdown', 'markdown.extensions'],
     python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*',
     install_requires=['setuptools >= 36'],
+    extras_require={
+        'testing': [
+            'coverage<5.0',
+            'pyyaml',
+            'pytidylib',
```

Comment:
Let's not have `pytidylib` here. Most users (and contributors) don't have the underlying c lib installed and the install of the Python lib will fail for them, which would prevent the tests from even beginning to run for them. `pytidylib` must be optional for everyone. We only enforce it's use on Travis.

Reply:
Haven't rechecked, but initially on Arch Linux without `tidy` being installed only some tests errored because of this, but installation (in tox) went fine.

Reply:
Yes, but what about users who run tests on Windows? They will fail every time if an attempt to install `pytidylib` happens. My point is that auto-installing `pytidylib` should only ever happen on Travis. If a user wants to run those tests locally, then they **must** manually install `pytidylib` as documented in the Contributing Guide.

Reply:
Well, it was previously installed always already (in tox).
I will remove it from "testing", but keep it in tox then.